deployer/cloudflaredeployer.go

```diff
@@ -0,0 +1,312 @@
+		return errors.Wrap(err, "failed to start")
+	}
+
+	api, err := getAPIClient()
+	if err != nil {
+		return errors.Wrap(err, "failed to initalize Cloudflare api client")
+	}
+
+	if err := os.RemoveAll(filepath.Join(ctx.Cwd, ".deployment")); err != nil {
+		if !os.IsNotExist(err) {
+			return errors.Wrap(err, "failed to RemoveAll deployment files")
+		}
+	}
+
+	if err := os.MkdirAll(filepath.Join(ctx.Cwd, ".deployment"), util.PermDirectory); err != nil {
+		return errors.Wrap(err, "failed to MkdirAll .deployment")
+	}
+
+	templatesPath, err := subotemplate.FullPath(job.repo, job.branch)
+	if err != nil {
+		return errors.Wrap(err, "failed to template.FullPath")
+	}
+
+	if job.updateTemplates {
+		templatesPath, err = subotemplate.UpdateTemplates(job.repo, job.branch)
+		if err != nil {
+			return errors.Wrap(err, "🚫 failed to UpdateTemplates")
+		}
+	}
+
+	for _, meta := range appSource.Applications() {
+		log.LogStart(fmt.Sprintf("deploying %s (%s) runnables as workers", meta.Identifier, job.domain))
+
+		runnablesByName := getRunnablesByName(appSource, meta)
+
+		workersToRoute := make(map[string]cfWorker)
+
+		for _, handler := range appSource.Handlers(meta.Identifier, meta.AppVersion) {
+			if handler.Input.Type != "request" {
+				log.LogWarn(fmt.Sprintf("unsupported handler type: %s; ignoring.", handler.Input.Type))
+				continue
+			}
+
+			if len(handler.Steps) != 1 {
+				log.LogWarn(fmt.Sprintf("only one step is currently supported per handler"))
+				continue
+			}
+
+			for _, step := range handler.Steps {
+				runnable, ok := runnablesByName[step.CallableFn.Fn]
+				if !ok {
+					return errors.Errorf("unexpected runnable %s", step.CallableFn.Fn)
+				}
+
+				script, err := renderRequestWorker(templatesPath, handler.Input.Method, handler.Input.Resource)
+				if err != nil {
+					return errors.Wrap(err, "failed to render request worker")
+				}
+				scriptName := fmt.Sprintf("%s-request-%s", strings.ReplaceAll(meta.Identifier, ".", "-"), runnable.Name)
+				if err := deployRunnable(api, &runnable, scriptName, script, job.domain); err != nil {
+					return errors.Wrap(err, "failed to deploy runnable")
+				}
+
+				path := handler.Input.Resource
+				workersToRoute[scriptName] = cfWorker{
+					Path:           path,
+					Method:         handler.Input.Method,
+					ServiceBinding: toWorkerBinding(scriptName),
+				}
+
+				log.LogInfo(fmt.Sprintf("deployed runnable %s", runnable.Name))
+			}
+		}
+
+		if err := deployRouter(templatesPath, api, workersToRoute, job.domain); err != nil {
+			return errors.Wrap(err, "failed to create router")
+		}
+		routerRoute := fmt.Sprintf("%s/*", job.domain)
+		if err := createRoute(api, routerRoute, routerScriptName, job.domain); err != nil {
+			log.LogWarn(fmt.Sprintf("error while creating route: %s", err))
+		}
+
+		log.LogDone(fmt.Sprintf("deployed to Cloudflare at %s", routerRoute))
+	}
+
+	return nil
+}
+
+func getAPIClient() (*cloudflare.API, error) {
+	apiToken := os.Getenv("CF_API_TOKEN")
```

Comment:
Could we make this `SUBO_CLOUDFLARE_TOKEN` ? Or is this a standard env key used by other tools?